### ISSUE TITLE
Form

### ISSUE THREAD

**Issue by balola** [mdelobelle/metadatamenu#79]
Hi,

I love your work and always think on it how to use.
I have an idea to use it as (database) form, like PDF form.
If I could use inline fields (more in one row) in rows, I could make contextual form to collect data.

> (prefix::{"type": "select", "options":["his", "her"]}) name is {{name}}, age: (age:: .. ) -- something like this mixture
> the fields come from fileClass or define in place
> the fields palceholders are UIs to make the process faster in view mode (like in dataviewjs)
> can build a process way, the order of UIs
> can print it
> can export the structured data to data folders (csv, excel, JSON, databases) 

It is very simple.
For you, not for me 😀.

What is your opinion?
Füles

**Comment by AtlcatlQ** [comment]
You can do that already, but all the results are saved in the same field: 

```
My Field:: {"type":"Input","options":{"template":"This is a template using a {{subField}} to get data"}}
````

You just have to go to `Manage <MyClass> fields`, select or add the field you want to set. Then, the field type should be "Accept any value". In "Template" you could set that the sub fields, I don't know if this is what you want.

Or, could set a template using templater plugin which allows place holders and can support a lot of successive inputs.

like: 

```
[pronouns:: <% tp.system.suggester(["his", "her", "their"], ["his", "her", "their"]) %>]
[name::  <% tp.system.prompt("Name of the person") %>]
[age::  <% tp.system.prompt("Age of the person") %>]
``` 

As they're inline fields you could use it even in a sentence that is the thing, I think, you want to. The only limitation is the inputs are requested in reverse order when are replaced. 

If this helps you let me know. But I don't know if I explained myself clearly. However I don't know how to make it using dataviewjs or exported to csv, excel, etc. I don't think any plugin could that. Have you think about using another app to achieve that, like microsoft Access or postgres, or google forms? (In case you need it in excels more than in notes)

**Comment by balola** [comment]
Hi,
Thanks your answere.
I have a HR Controller, but I write programs to help my work.
I used Access, MS SQL, flat databases, wrote programs in basic, PHP, Python, C# and ...
I have just looked at your program and realised how fast I could make collection **forms** based on database **like** system.
Just for myself I put the documentum system on Obsidian and it could help me to fill documents (forms) and collect information from them and your solution seems very practical. 😀
Füles
